spec/requests/api/v1/users/create_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/', type: :request do
+  let(:user)            { User.last }
+  let(:failed_response) { 422 }
+  let(:internal_server_error) { 500 }
+
+  describe 'POST create' do
+    let(:email)                 { 'test@test.com' }
```

Comment:
You can take adventage of the usage of a factory. Check out the [docs](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md)

Basically you should define a factory for the `User` model and then use `attributes_for` method which returns a hash with attributes to be used on a instance. For example if you want to use it with the user's factory:
```
attributes_for(:user)
```